tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Info_b2fff8.yaml

```diff
@@ -0,0 +1,132 @@
+_file_name: Course_Info_b2fff8.yaml
+cache_timeout: null
+certification_details: null
+certified_by: null
+dataset_uuid: 27a4476b-5a9a-4fc4-ac75-da520b522341
+description: null
+params:
+  adhoc_filters:
+  - clause: WHERE
+    comparator: No filter
+    expressionType: SIMPLE
+    operator: TEMPORAL_RANGE
+    subject: emission_time
+  all_columns: []
+  color_pn: false
+  column_config:
+    active_count:
+      columnWidth: 200
+    course_name:
+      columnWidth: 300
+    enrollees:
+      columnWidth: 150
+    org:
+      columnWidth: 200
+    tag_list:
+      truncateLongCells: true
+  conditional_formatting: []
+  extra_form_data: {}
+  groupby:
+  - course_name
+  - org
+  metrics:
+  - enrollees
+  - active_count
+  - tag_list
+  order_by_cols: []
+  order_desc: false
+  percent_metrics: []
+  query_mode: aggregate
+  row_limit: 50000
+  server_page_length: 10
+  show_cell_bars: false
+  table_timestamp_format: smart_date
+  temporal_columns_lookup: {}
+  time_grain_sqla: P1M
+  viz_type: table
+query_context:
+  datasource:
+    id: 221
+    type: table
+  force: false
+  form_data:
+    adhoc_filters:
+    - clause: WHERE
+      comparator: No filter
+      expressionType: SIMPLE
+      operator: TEMPORAL_RANGE
+      subject: emission_time
+    all_columns: []
+    color_pn: false
+    column_config:
+      active_count:
+        columnWidth: 200
```

Comment:
These aren't usually present, and seem to keep the columns from resizing when the screen is smaller. Any idea why they're here?

Reply:
yes, that was a conscious decision to prevent the 'tags' column from taking over the whole space and squishing the rest of the columns together

Reply:
it will, but if we dont specify the width for the other columns, itll still push all the columns to the left 
![image](https://github.com/user-attachments/assets/20b98426-6f57-45c5-b620-02253f588ea1)

Reply:
Hmm yeah I see. 

@Ian2012 can we get the course comparison dashboard embedded on the sandbox site?
@crathbun428 do we have any guidance for the resolutions / page sizes we want to support?

I know we're not currently worrying about mobile, but I think we may need to adjust the sizes to make things fit reasonably on, say, a laptop screen when embedded. Does that sound fair?

Reply:
This does sound fair - I did not think about this, but can come up with some requirements for this ASAP.

Reply:
For sure, I think this is non-blocking for this PR and we can clean up the specific sizes later.